db/version_set.cc

```diff
@@ -1574,7 +1577,7 @@ VersionStorageInfo::VersionStorageInfo(
     const InternalKeyComparator* internal_comparator,
     const Comparator* user_comparator, int levels,
     CompactionStyle compaction_style, VersionStorageInfo* ref_vstorage,
-    bool _force_consistency_checks)
+    bool _force_consistency_checks, bool _dynamic_level_bytes)
```

Comment:
seems no need to add a leading `_`

Reply:
some compilers will complain about this, and that's why `force_consistency_checks` with a leading `_`

Reply:
I thought that's because we have a member function named `force_consistency_checks`? But we don't have that getter for dynamic bytes.

Reply:
oh, you're right.